doc/classes/AudioEffectSampleRate.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioEffectSampleRate" inherits="AudioEffect" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Adds a sample rate audio effect to an Audio bus.
+	</brief_description>
+	<description>
+		Reduces the input audio's sample rate, without interpolating the output. Also known as "sample & hold", or a type of "bitcrush".
```

Comment:
Escape the ampersand character (XML requirement):

```suggestion
		Reduces the input audio's sample rate, without interpolating the output. Also known as "sample &amp; hold", or a type of "bitcrush".
```

Reply:
Fixed!